tf2_ros/include/tf2_ros/static_transform_broadcaster.h

```diff
@@ -54,7 +54,9 @@ class StaticTransformBroadcaster{
 
   /** \brief Send a TransformStamped message
    * The stamped data structure includes frame_id, and time, and parent_id already.  */
-  void sendTransform(const geometry_msgs::TransformStamped & transform);
+  void sendTransform(const geometry_msgs::TransformStamped & transform) {
```

Comment:
This simple API transformation might be worth an inline function.

Reply:
Due to changing the ABI here. I'm going to retarget this for noetic